app/design/adminhtml/default/default/template/cms/browser/tree.phtml

```diff
@@ -60,6 +60,8 @@
     });
 
     tree.expandPath('<?= $this->getTreeCurrentPath() ?>').then((node) => {
-        MediabrowserInstance.selectFolder(node);
+        MediabrowserInstance.selectFolder(node, true).then(() => {
+            MediabrowserInstance.selectFileById('<?= $this->getTreeCurrentFile() ?>');
+        });
```

Comment:
@fballiano this is the code that should select the correct folder and file in case you want to debug it. Could it just have been a browser cache issue?

Reply:
mmm no, also testing after a hard reload I can't make it work :-\

Reply:
Hm, maybe something when I did the path in session stuff. I'll definitely make it work again though.